server/internal/graphql/resolver/meshsync.go

```diff
@@ -34,58 +34,72 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
 			mesherydbPath := path.Join(utils.GetHome(), ".meshery/config")
 			err := os.Mkdir(path.Join(mesherydbPath, ".archive"), os.ModePerm)
 			if err != nil && os.IsNotExist(err) {
-				return "", err
+				createFolder := models.ErrMakeDir(err, path.Join(mesherydbPath, ".archive"))
```

Comment:
Instead of creating those error in a different package. Meshkit errors used in a package are created in the same package for clear namespacing. Please create those errors in this file https://github.com/meshery/meshery/blob/master/server/internal/graphql/resolver/error.go